src/components/Rules/styled.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+export const Rule = styled.div`
+  z-index: 10;
+  font-family: Arial, Helvetica, sans-serif;
+  position: fixed;
+  top: 5%;
+  bottom: 5%;
+  right: 5%;
+  left: 5%;
+  margin: auto;
+  max-width: 80ch;
+  border-radius: 25px;
+  border: 2px outset var(--red);
+  background: var(--red);
```

Comment:
Trying this on my own machine, I realized that the red can be very overwhelming. Maybe we can just do black and white with red and/or blue headers?

Reply:
Trying different combos. Will resolve when we decide on something nice.